_posts/2023-04-19-12.0.0-release.md

```diff
@@ -191,8 +205,6 @@ implementation, see the latest [Arrow Rust changelog][5].
 [GH-21429]: https://github.com/apache/arrow/issues/21429
 [GH-21430]: https://github.com/apache/arrow/issues/21430
 [GH-33243]: https://github.com/apache/arrow/issues/33243
-[GH-33819]: https://github.com/apache/arrow/issues/34819
-[GH-33837]: https://github.com/apache/arrow/issues/34837
```

Comment:
My markdown formatter removed these links but it appears it did so because they are not referenced anywhere so it should be safe to remove.